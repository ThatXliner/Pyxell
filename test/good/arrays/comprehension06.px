
a = [(x, y) for x in 1..10 step 2 if x > 3 for y in x...0 step -3 if y < 10]
for x in a do
    print x
