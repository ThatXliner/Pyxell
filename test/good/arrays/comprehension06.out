(5, 5)
(5, 2)
(7, 7)
(7, 4)
(7, 1)
(9, 9)
(9, 6)
(9, 3)
